chrome-extension/content.js

```diff
@@ -196,10 +220,15 @@ async function addItem() {
       throw new Error(response.statusMessage);
     } else {
       // TODO: display success message, timeOut closeSidebar
-      closeSidebar();
+      sidebar.removeChild(sidebar.childNodes[1]);
```

Comment:
If this is removing the form, I think it would be better to get that element specifically by ID or class or tag because it could be easy to forget to change this if you end up adding a different 2nd element to the sidebar later.

Reply:
ah yes, fixed